blog/modules/ROOT/pages/testing-in-pkl.adoc

```diff
@@ -240,9 +251,11 @@ For intentional changes, add the `--overwrite` flag. This will overwrite the exp
 
 [source]
 ----
-$ pkl test person.pkl --overwrite
```

Comment:
Why remove the command line?

Reply:
It does seem inconsistent across examples; the only other one that includes(/d) the command line is `pkl test math.pkl`. I _think_ I'd prefer to add the command line to all of them, instead of removing this one, but I'm 51/49 on this.

Reply:
Yeah. Agree with the all or nothing approach.

Reply:
Mistake! Adding this back.